include/co_api.h

```diff
@@ -223,11 +223,19 @@ typedef enum co_dtype
 /** Access function event */
 typedef enum od_event
 {
-   OD_EVENT_READ,    /**< Read subindex */
-   OD_EVENT_WRITE,   /**< Write subindex */
-   OD_EVENT_RESTORE, /**< Restore default value */
+   OD_EVENT_READ,       /**< Read subindex */
+   OD_EVENT_WRITE,      /**< Write subindex */
+   OD_EVENT_RESTORE,    /**< Restore default value */
 } od_event_t;
 
+/** Notify event */
+typedef enum od_notify_event
+{
+   OD_NOTIFY_ACCESSED,
+   OD_NOTIFY_VALUE_SET,
+   OD_NOTIFY_SDO_RECEIVED,
```

Comment:
Are both ACCESSED and VALUE_SET required? They are both called from co_od_set_value and it seems that the only difference is whether the user has specified that the object has an accessor function or not. 

SDO_RECEIVED seems to be used when an object larger than 64 bits is being downloaded and will be called for every partial update with the size received so far. So the value in this case is a size. This should be documented. There should also be a test that verifies the functionality.